apilos_settings/api/tests/test_apis.py

```diff
@@ -42,3 +44,89 @@ def test_get_config_route(self):
             "version": "0.0",
         }
         self.assertEqual(response.data, expected)
+
+
```

Comment:
Ceci annule peut-être ma remarque précédente ☝️ mais est-ce qu'ici tu ne gagnerais pas à  ajouter aussi:

```suggestion
@override_settings(USE_MOCKED_SIAP_CLIENT=True)
```

Reply:
done !